Modules/_ctypes/callbacks.c

```diff
@@ -295,6 +281,14 @@ static void _CallPythonObject(ctypes_state *st,
     for (j = 0; j < i; j++) {
         Py_DECREF(args[j]);
     }
+    return;
+
+  Error:
+    PyErr_FormatUnraisable(
+            "Exception ignored on creating argument %zd for "
```

Comment:
```suggestion
            "Exception ignored while creating argument %zd for "
```

Reply:
See also error messages on lines 258 and 271: "Exception ignored on calling ctypes callback function" and "Exception ignored on converting result of ctypes callback function". They were initially introduced in 10228bad0452d94e66c964b625a0b61befa08e59.